### DISCUSSION TITLE
Problema integrazione condizionatore tradizionale tramite Broadlink e SmartIR

### DISCUSSION THREAD

**Discussion by chip83** [indomus/forum#2988] [HUB: Home Assistant]
Ho seguito [questa guida](https://indomus.it/progetti/domotizzare-un-condizionatore-tradizionale-con-broadlink-e-home-assistant-v2/) per cercare di far funzionare il mio condizionatore tramite smartIR.
Ho creato un file json relativo ai codici del mio condizionatore (Panasonic CS-UE9PKE). Il file è stato validato e risulta "formalmente ok". I codici li ho acquistiti tramite Broadlink Manager (e inviandoli tramite il suddetto programma producono il loro effetto sul condizionatore).
Ho inserito l'interfaccia su HA che vedo correttamente con le varie temperature e modalità di funzionamento.
Il problema è che non funziona! :D 
Qualsiasi comando do al condizionatore (tramite l'interfaccia grafica) non produce nessun effetto.

![immagine](https://github.com/indomus/forum/assets/22590434/7e7d7907-e63b-4ca4-979c-14400f6f83d8)

Qui ci sono da fare due considerazioni:
- il mio condizionatore ha un codice diverso tra ON e OFF. Nel file json ho inserito solo il codice OFF (come da guida)
- se invio un qualsiasi comando tramite il telecomando il condizionatore produre un bip per notificarne della ricezione (anche se non produce alcun effetto se il condizionatorre è spento). Se invio un comando tramite HA non succede proprio nulla (nessun bip).

Premetto che ho assegnato un IP statico al broadlink che è lo stesso inserito nel file di configurazione.

Cosa potrebbe essere?

**Comment by suibaf** [comment]
dai qualche elemento in più, il file di confugurazione.

**Comment by macuser79** [comment]
Se posti il file di configurazione ti si può dare una mano, anche io ieri ho avuto il tuo stesso problema ma poi alla fine sono riuscito a farlo andare.

**Comment by chip83** [comment]
[6116.json](https://github.com/indomus/forum/files/14955738/6116.json)

scusate, ho avuto un po' da fare in questo periodo.
Questo è il file che ho fatto.

**Comment by chip83** [comment]
qualcuno può darmi qualche dritta?
